pkgs/tools/security/sbomnix/default.nix

```diff
@@ -1,53 +1,85 @@
-{ lib
-, fetchFromGitHub
-, coreutils
-, curl
-, gnugrep
-, gnused
-, gzip
-, nix
-, python
+{
+  lib,
+  fetchFromGitHub,
+  grype,
+  nix,
+  nix-visualize,
+  python,
+  vulnix,
   # python libs
-, colorlog
-, graphviz
-, numpy
-, packageurl-python
-, pandas
-, requests
-, reuse
-, tabulate
+  beautifulsoup4,
+  colorlog,
+  dfdiskcache,
+  graphviz,
+  numpy,
+  packageurl-python,
+  packaging,
+  pandas,
+  pyrate-limiter,
+  requests,
+  requests-cache,
+  requests-ratelimiter,
+  reuse,
+  tabulate,
 }:
-
 python.pkgs.buildPythonApplication rec {
   pname = "sbomnix";
-  version = "1.4.5";
+  version = "1.6.1";
 
   src = fetchFromGitHub {
     owner = "tiiuae";
-    repo = pname;
+    repo = "sbomnix";
     rev = "refs/tags/v${version}";
-    hash = "sha256-RxDFxVGivVBw2uhtzf231Q2HHTBFKSqGrknr2Es/ygM=";
+    hash = "sha256-kPjCK9NEs3D0qFsSSVX6MYGKbwqeij0svTfzz5JC4qM=";
+
+    # Remove documentation as it contains references to nix store
+    postFetch = ''
+      rm -fr "$out"/doc
+      find "$out" -name '*.md' ! -name "README.md" -exec rm -f '{}' \;
+    '';
   };
 
-  makeWrapperArgs = [
-    "--prefix PATH : ${lib.makeBinPath [ coreutils curl gnugrep gnused gzip graphviz nix ]}"
-  ];
+  postInstall = ''
+
+    wrapProgram $out/bin/sbomnix \
+        --prefix PATH : ${lib.makeBinPath [nix graphviz]}
+
+    wrapProgram $out/bin/nixgraph \
+        --prefix PATH : ${lib.makeBinPath [nix graphviz]}
+
+    wrapProgram $out/bin/vulnxscan \
+        --prefix PATH : ${lib.makeBinPath [grype nix vulnix]}
+
+    wrapProgram $out/bin/nix_outdated \
+        --prefix PATH : ${lib.makeBinPath [nix-visualize]}
+
+    wrapProgram $out/bin/provenance \
+        --prefix PATH : ${lib.makeBinPath [nix]}
+
+  '';
```

Comment:
```suggestion
  postInstall = ''
    wrapProgram $out/bin/sbomnix \
        --prefix PATH : ${lib.makeBinPath [nix graphviz]}
    wrapProgram $out/bin/nixgraph \
        --prefix PATH : ${lib.makeBinPath [nix graphviz]}
    wrapProgram $out/bin/vulnxscan \
        --prefix PATH : ${lib.makeBinPath [grype nix vulnix]}
    wrapProgram $out/bin/nix_outdated \
        --prefix PATH : ${lib.makeBinPath [nix-visualize]}
    wrapProgram $out/bin/provenance \
        --prefix PATH : ${lib.makeBinPath [nix]}
  '';
```
I would love to combine those but I am not sure how. We could say we wrap all program with the same PATH since they are not in seperate outputs and we can't save any closure size here anyway

Reply:
Fixed the formatting as you suggested in https://github.com/NixOS/nixpkgs/pull/299429/commits/b70f1456dbff1d409d6bfdac072b87b52456c0d0.
I'm also not sure how to combine them the way you suggested, let me know if I should change this somehow.

Reply:
something like

```
  postInstall = ''
    for program in sbomnix nixgraph vulnxscan nix_outdated provenance; do
      wrapProgram $out/bin/sbomnix \
        --prefix PATH : ${lib.makeBinPath [nix nix-visualize graphviz grype vulnix]}
    done
  '';
```

Reply:
Hnmh, I'd personally prefer keeping these separate. It makes it easier to split things up, and you don't need to read the source anymore to figure out what's shelling out to what.

Reply:
I would tend to agree with @flokli here. 
@SuperSandro2000: let me know if you insist having this the way you proposed and I'll change it.